### PR TITLE
docs: add user guide on Decibels app 

### PR DESCRIPTION
**Pull request by CBID2**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

> [!NOTE]
> This PR adds a user guide about the ~~Audio Sharing~~ Decibels app in the GNOME section. I decided to create this as a replacement for PR #2249 to satisfy the requirements for RSOD. 

#### Issue
Relates to #2161 

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

**Comment by CBID2** [comment]
Ready for your review @wsoyinka

File: docs/desktop/gnome/audio-sharing.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+This guide assumes you have the following:
+
+- Rocky Linux
+- Flatpak
+- FlatHub
+
+## Installation Process
+
+Go to Flatub.com, type "Audio Sharing" in the search bar, and click on **Install**.
```
Comment (wsoyinka) [inline docs/desktop/gnome/audio-sharing.md:21]: flathub.org not flatub.com

Comment (CBID2) [reply docs/desktop/gnome/audio-sharing.md]: Made the change @wsoyinka

**Comment by wsoyinka** [comment]
Thank you for this @CBID2   !

To make this submission a bit more complete (and mergable) , please investigate and include the following:

- Original screenshots different from the ones readily available on the app website
- An example walk-through demonstrating playing a sample audio file/source from the system running the "audio sharing app". .e.g. a lengthy youtube video/audio 
 - Corresponding simple example showing the reader how to connect to the stream from a separate device. Perhaps using VLC as a sample client on a different LAN device
- Explicit instructions for how to punch a hole in the host firewall (e.g. firewalld) to make the stream accessible to other LAN systems/users

**Comment by CBID2** [comment]
> Thank you for this @CBID2   !
> 
> 
> 
> To make this submission a bit more complete (and mergable) , please investigate and include the following:
> 
> 
> 
> - Original screenshots different from the ones readily available on the app website
> 
> - An example walk-through demonstrating playing a sample audio file/source from the system running the "audio sharing app". .e.g. a lengthy youtube video/audio 
> 
>  - Corresponding simple example showing the reader how to connect to the stream from a separate device. Perhaps using VLC as a sample client on a different LAN device
> 
> - Explicit instructions for how to punch a hole in the host firewall (e.g. firewalld) to make the stream accessible to other LAN systems/users
> 
> 

Hi @wsoyinka. I spoke to one of the admins of the Audio Sharing community server to get some ideas to implement your suggestions. He mentioned that there is nothing more to show except that users start the app and the QR code appears. With this in mind, I believe the best option would leave the guide as is.

**Comment by wsoyinka** [comment]
Thank you for looking further into this @CBID2 !

Do you have a Rocky Linux desktop system yourself to try getting the app to work out of the box ?  

If so perhaps you can document the procedure that you went through on the "app" side and the "client" side with your own screenshot screenshots.

**Comment by wsoyinka** [comment]
Understood @CBID2 .  No worries.

If you had a Rocky Linux system or similar to work/test with it would have been easy for to you to walk-through the procedure yourself and be able to authoritatively document it for others. 

The apps works as intended but some assumptions are made about the end user's environment. 

For example at minimum the default port 8554/TCP (rtsp-alt) port needs to be accessible/open on the system running the app so that LAN users/clients can connect to it.   There is little utility if other devices cannot connect to the shared stream.   

We'll see about merging your contribution as-is - with an aim to improving it in the future.

Hopefully, you'll be able to get a Rocky Linux VM running to assist with writing and testing any remaining RSOD contributions.

Many thanks again ❤️ !

**Comment by CBID2** [comment]
> Understood @CBID2 .  No worries.
> 
> 
> 
> If you had a Rocky Linux system or similar to work/test with it would have been easy for to you to walk-through the procedure yourself and be able to authoritatively document it for others. 
> 
> 
> 
> The apps works as intended but some assumptions are made about the end user's environment. 
> 
> 
> 
> For example at minimum the default port 8554/TCP (rtsp-alt) port needs to be accessible/open on the system running the app so that LAN users/clients can connect to it.   There is little utility if other devices cannot connect to the shared stream.   
> 
> 
> 
> We'll see about merging your contribution as-is - with an aim to improving it in the future.
> 
> 
> 
> Hopefully, you'll be able to get a Rocky Linux VM running to assist with writing and testing any remaining RSOD contributions.
> 
> 
> 
> Many thanks again ❤️ !
> 
>  

Thanks @wsoyinka. I also updated my proposal. I’d love your feedback on it

**Comment by CBID2** [comment]
Hey @wsoyinka! :) I managed to get Rocky Linux to work on VirtualBox on a different computer. Sadly, the audio sharing app did not work there(the app's admin mentioned that the QR code does not work on virtual machines). So, to save time, I decided to replace the content with information about the Decibels app instead.

**Comment by CBID2** [comment]
Hey @wsoyinka. Are you going to review this?

**Review by wsoyinka — APPROVED** [review]

**Comment by wsoyinka** [comment]
> Hey @wsoyinka. Are you going to review this?

Hey @CBID2  ....  Anybody in the community can review contributions submitted by others if they have the spare cycles ! 
You (@CBID2 ) are welcome and encouraged to also review/test/improve the contributions submitted by others too :-)  

I happen to have some cycles now and I just reviewed the  "user guide on Decibels app" and it can be merged. 

Many thanks for working on it and contributing it 😄 🥳  !!!

**Comment by CBID2** [comment]
> > Hey @wsoyinka. Are you going to review this?
> 
> 
> 
> Hey @CBID2  ....  Anybody in the community can review contributions submitted by others if they have the spare cycles ! 
> 
> You (@CBID2 ) are welcome and encouraged to also review/test/improve the contributions submitted by others too :-)  
> 
> 
> 
> I happen to have some cycles now and I just reviewed the  "user guide on Decibels app" and it can be merged. 
> 
> 
> 
> Many thanks for working on it and contributing it 😄 🥳  !!to it 😄 🥳  !!!

Thanks @wsoyinka! 
I finished the user guide for the Decoder app, so now I'm going to submit my PR for it.

**Pull request merged**